charger/ocpp.go

```diff
@@ -146,9 +150,16 @@ func NewOCPP(id string, connector int, idtag string,
 		ocpp.KeyMeterValuesSampledData,
 		ocpp.KeyMeterValueSampleInterval,
 		ocpp.KeyConnectorSwitch3to1PhaseSupported,
+		ocpp.KeyChargingScheduleAllowedChargingRateUnit,
 	}
 	_ = keys
 
+	if chargingRateUnit == "W" {
```

Comment:
```suggestion
	c.chargingRateUnit = types.ChargingRateUnitType(chargingRateUnit)
```

Reply:
Da fehlt mir die Go-Erfahrung. In Java sind das völlig verschiedene Dinge.